app/build.gradle

```diff
@@ -18,46 +18,39 @@ repositories {
 
 dependencies {
     compile 'com.google.code.gson:gson:2.3'
-
     // Dependency Injection
     compile 'com.squareup.dagger:dagger:1.0.1'
     compile 'com.squareup.dagger:dagger-compiler:1.0.1'
     compile 'com.jakewharton:butterknife:6.1.0'
-
     compile 'com.actionbarsherlock:viewpagerindicator:2.4.1'
     compile 'com.nineoldandroids:library:2.4.0'
     compile 'com.github.kevinsawicki:wishlist:0.9'
-
     // Event Bus
     compile 'com.squareup:otto:1.3.5'
-
     // Image Loading
     compile 'com.squareup.picasso:picasso:2.3.2'
     compile 'com.nostra13.universalimageloader:universal-image-loader:1.9.3'
-
     // HTTP
     compile 'com.squareup.okhttp:okhttp-urlconnection:2.0.0'
     compile 'com.squareup.okhttp:okhttp:2.3.0'
     compile 'com.squareup.retrofit:retrofit:1.9.0'
-
     // Android Support Libs
     compile 'com.android.support:support-v4:22.0.0'
     compile 'com.android.support:appcompat-v7:22.0.0'
-
     //SlidingUp Panel
     compile 'com.sothree.slidinguppanel:library:3.0.0'
-
     compile 'com.joanzapata.android:android-iconify:1.0.9'
     compile 'com.astuetz:pagerslidingtabstrip:1.0.1'
-
```

Comment:
Why empty lines removed? They are there to not to make this file look cluttered. Please put it back.